File: translation_output/misleading_test/gpt-4/codenet/java/python-sanitized/p02251.py
class p02251:
    def solve(self, n):
        ans = 0
        
        # 25 cents
        ans += n // 25
        n %= 25
        
        # 10 cents
        ans += n // 10
        n %= 10
        
        # 5 cents
        ans += n // 5
        n %= 5
        
        # 1 cent
        ans += n

        return ans

# Test input
test_input = 100

# Expected output: 82
solution = p02251()
print(solution.solve(test_input))

File: Results/DER/Translation/translation_output/misleading_test/deepseekcoder/codenet/python/java/p02251.java
<｜begin▁of▁sentence｜>You are an expert Python programmer.Your task is to write a Python function to solve a programming problem.

### Instruction:
Translate the following python code to java and enclose your solution inside ```java```.
A sample test case is provided below:

Test input:
100

Expected output:
25

```
n = int(input())
ans = 0
for x in (25, 10, 5, 1):
  ans += n // x
  n %= x
print(ans)

```

### Response:
Here is the Java version of the Python code:

```java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        int ans = 0;
        for (int x : new int[]{25, 10, 5, 1}) {
            ans += n / x;
            n %= x;
        }
        System.out.println(ans);
    }
}
```

This Java code does the same thing as the Python code. It reads an integer from the input, then it divides it by 25, 10, 5, and 1 in order, and adds the quotient to `ans`. The remainder of the division is then assigned back to `n`. Finally, it prints the value of `ans`.
<|EOT|>
